crabs/detection_tracking/models.py

```diff
@@ -0,0 +1,157 @@
+import os
+import pickle
+import tempfile
+import time
+from datetime import datetime
+
+import mlflow
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torchvision
+from torch.utils.data import DataLoader
+from torch.utils.tensorboard import SummaryWriter
+from torchvision.models.detection.faster_rcnn import FastRCNNPredictor
+
+
+def create_faster_rcnn(num_classes: int, coco_model: bool = True) -> nn.Module:
+    """Create faster rcnn model for the training
+
+    Parameters
+    ----------
+    num_classes : int
+        number of classes to train
+    coco_moder : bool
```

Comment:
```suggestion
    coco_model : bool
```
I thought this meant whether the annotations will be in COCO format or not, but I think you mean whether to get the faster RCNN model with all the COCO classes as output, or with custom classes. Is that correct?

If so, maybe we could just have num_classes equal to None by default, and if it's None then we return the faster RCNN model with all the COCO classes.